openshift/release/release-notes.md

```diff
@@ -0,0 +1,376 @@
+
+* :sparkles: Generate OpenAPI specification and Swagger file ([#3081](https://github.com/tektoncd/pipeline/pull/3081))
+
+* :sparkles: Leverage logstream to get better e2e diagnostic logs. ([#3388](https://github.com/tektoncd/pipeline/pull/3388))
+
+* :sparkles: Add ability to replace image names for examples tests based on arch ([#3299](https://github.com/tektoncd/pipeline/pull/3299))
+
+## Fixes
+* :bug: Adjust PipelineRun's StartTime based on TaskRun state. ([#3461](https://github.com/tektoncd/pipeline/pull/3461))
+  
+  Fixes a bug where PipelineRun may report as Failed when it really timed out.
+
+* :bug: Make ConditionCheck container names DNS-safe ([#3394](https://github.com/tektoncd/pipeline/pull/3394))
+
+  Fixes a bug where condition validation may fail with long condition names.
+
+* :bug: Don't fail immediately on missing resources. ([#3385](https://github.com/tektoncd/pipeline/pull/3385))
+  
+  Add a grace period for resources to appear before failing *Runs
+
+* :bug: Switch webhook liveness/readiness probes to use http ports ([#3349](https://github.com/tektoncd/pipeline/pull/3349))
+
+* :bug: Attempt to fix another structured-merge-diff failures. ([#3348](https://github.com/tektoncd/pipeline/pull/3348))
+
+  action required: Drop pointers from inline struct when embedding TaskSpec in v1beta1
+
+* :bug: avoid requeuing taskrun in case of permanent error ([#3068](https://github.com/tektoncd/pipeline/pull/3068))
+  
+  Do not requeue taskrun if it was rejected with permanent error. This bug was causing the incorrect metrics for tekton_taskrun_count{status="failed"}.
+
+* :bug: Check that events are at least expected ones ([#3375](https://github.com/tektoncd/pipeline/pull/3375))
+* :bug: Eliminate redundant Pod Get. ([#3496](https://github.com/tektoncd/pipeline/pull/3496))
+* :bug: tests: fix potential races with t.Parallel and loops loop ([#3493](https://github.com/tektoncd/pipeline/pull/3493))
+* :bug: Fix error path through test. ([#3474](https://github.com/tektoncd/pipeline/pull/3474))
+* :bug: fix finally e2e test ([#3464](https://github.com/tektoncd/pipeline/pull/3464))
+* :bug: Don't "Follow" when dumping test pod logs. ([#3447](https://github.com/tektoncd/pipeline/pull/3447))
+* :bug: Make t.Parallel() come first ([#3425](https://github.com/tektoncd/pipeline/pull/3425))
+* :bug: release: fix the publish task definition bookmark_tabs ([#3376](https://github.com/tektoncd/pipeline/pull/3376))
+* :bug: Update the revision used with kaniko. ([#3363](https://github.com/tektoncd/pipeline/pull/3363))
+* :bug: Pass test ctx to taskrun reconciler instead of background ctx ([#3445](https://github.com/tektoncd/pipeline/pull/3445))
+
+## Misc
+* :hammer: Put Tekton OCI bundles behind a feature flags flags ([#3492](https://github.com/tektoncd/pipeline/pull/3492))
+
+  Introduce a feature-flag enable-tekton-oci-bundle to be able to use Tekton Bundle through the pipeline API
```

Comment:
Related to the "Tekton Bundle" above, because it's disabled by default, we could not talk about it at all